pandas/io/parsers/readers.py

```diff
@@ -242,11 +242,6 @@
     :ref:`io.csv.mixed_timezones` for more.
 
     Note: A fast-path exists for iso8601-formatted dates.
-infer_datetime_format : bool, default False
```

Comment:
Shouldn't this technically be left in the docstring since it's only deprecated?

Reply:
you're right - done